composer.json

```diff
@@ -89,7 +89,13 @@
         "symfony/http-kernel": "If you want to use HttpRequestVerifier from symfony's bridge",
         "symfony/http-foundation": "If you want to use HttpRequestVerifier or HttpResponse reply from symfony's bridge",
         "symfony/form": "If you want to use forms",
-        "monolog/monolog": "In case you want to use PSR-3 logger"
+        "monolog/monolog": "In case you want to use PSR-3 logger",
+        "ajbdev/authorizenet-php-api": "If you want to use Authorizenet.net",
```

Comment:
`If you want to use Authorizenet.NET install ajbdev/authorizenet-php-api:1.1.* library`